core/server/services/routing/controllers/static.js

```diff
@@ -7,6 +7,13 @@ function processQuery(query, locals) {
     const api = require('../../../api')[locals.apiVersion];
     query = _.cloneDeep(query);
 
+    // Include tags and authors as well in case the resource is a `post`
+    if (_.get(query, 'resource') === 'posts') {
+        _.extend(query.options, {
```

Comment:
This will override any user configurations. The user can configure `include` in the `data` key.

We need to apply them as "defaults" and not override.